### DISCUSSION TITLE
Call to instantiated class

### DISCUSSION THREAD

**Discussion by nopsled** [vfsfitvnm/frida-il2cpp-bridge#248] [Q&A] [answered]
<img width="660" alt="image" src="https://user-images.githubusercontent.com/78918436/211953546-1e3a1209-216b-4bba-a7eb-40463739b89e.png">

Trying to call to SoundManager (game in Unity) and the class is not static. Therefore I am using the heap to locate the instance but at line which contains the invoke the game crashed on iOS. How can I solve this method calling?

**Comment by berkedel** [comment, marked as answer]
Before invoking the `ChangeMusic` method, have you checked the `obj` value was not null?
